torch/quantization/QConfig.py

```diff
@@ -63,3 +65,25 @@ def __new__(cls, weight):
 
 default_qat_qconfig = QConfig(activation=default_fake_quant,
                               weight=default_weight_fake_quant)
+
+def get_default_qconfig(backend='fbgemm'):
```

Comment:
technically - it's not even fbgemm, but rather fbgemm on per-CooperLake x86 :-\ . But I don't know how to make it nicer.

Reply:
I agree, perhaps just call it server and have a comment stating that you need to know about the hardware platform to ensure you are picking a good default?

Reply:
Actually with mkl-dnn not sure how to improve on this, so leaving this as is for now